website/docs/r/is_instance_group.html.markdown

```diff
@@ -69,7 +69,7 @@ The following arguments are supported:
 
 * `name` - (Required, Forces new resource, string) The name of the instance group.
 * `instance_template` - (Required, Forces new resource, string) The ID of the instance template to create the instance group.
-* `instance_count` - (Required, int) The number of instances to be created under the instance group.
+* `instance_count` - (Optional, int) The number of instances to be created under the instance group. Default is set to 0.
```

Comment:
Add a comment here saying the instance_count can be updated only when the manager is disabled